src/components/DocPage/DocPage.tsx

```diff
@@ -152,12 +166,20 @@ class DocPage extends React.Component<DocPageInnerProps, DocPageState> {
         } = this.props;
 
         const hideMiniToc = !this.showMiniToc;
+        const hideBurger = typeof headerHeight !== 'undefined' && headerHeight !== 0;
```

Comment:
probably undefined check is not needed here

Reply:
A variable can be undefined, so before checking its numeric value, we need to make sure that it is not undefined. But I will replace the headerHeight !== 0 on headerHeight > 0, because the height cannot be less than zero